op-node/rollup/attributes/engine_consolidate.go

```diff
@@ -125,13 +135,32 @@ func checkEIP1559ParamsMatch(opCfg *params.OptimismConfig, attrParams *eth.Bytes
 			translated = true
 		}
 
-		bd, be := eip1559.DecodeHoloceneExtraData(blockExtraData)
+		// Decode block parameters and check for mismatch
+		var (
+			bd, be   uint64
+			bm       uint8
+			extraErr string
+		)
+		if isJovian {
+			bd, be, bm = eip1559.DecodeMinBaseFeeExtraData(blockExtraData)
+		} else {
+			bd, be = eip1559.DecodeHoloceneExtraData(blockExtraData)
+		}
+
 		if ad != bd || ae != be {
-			extraErr := ""
 			if translated {
-				extraErr = " (translated from 0,0)"
+				if isJovian {
+					extraErr = " (translated from 0,0,0)"
```

Comment:
The third value (minimum base fee) is never translated from zero because there was never a protocol constant for minimum base fee. So `extraErr` shouldn't need to be modified for Jovian. We just need to make sure the Jovian error message separates the EIP 1559 params (with extraErr) from the minimum base fee, which extraErr does not apply to.